src/main/java/g0301_0400/s0382_linked_list_random_node/readme.md

```diff
@@ -13,13 +13,18 @@ Implement the `Solution` class:
 
 ![](https://assets.leetcode.com/uploads/2021/03/16/getrand-linked-list.jpg)
 
-**Input** \["Solution", "getRandom", "getRandom", "getRandom", "getRandom", "getRandom"\] \[\[\[1, 2, 3\]\], \[\], \[\], \[\], \[\], \[\]\]
+**Input** 
 
-**Output:** \[null, 1, 3, 2, 2, 3\]
+    \["Solution", "getRandom", "getRandom", "getRandom", "getRandom", "getRandom"\] 
+    \[\[\[1, 2, 3\]\], \[\], \[\], \[\], \[\], \[\]\]
```

Comment:
![image](https://user-images.githubusercontent.com/348496/146641925-50a494e6-a4e0-4148-99d5-a28169d8691d.png)

We need to reoemove escap for blocks.